packages/grid/src/features/standard-table/hooks/UseCellBackground.ts

```diff
@@ -0,0 +1,27 @@
+import { useMemo } from "react";
+import { StandardTableColumnConfig } from "../config/StandardTableConfig";
+import { useColumnFromConfig } from "./UseColumnFromConfig";
+
+const useBackground = <TItem>(
+  backgroundResolver: ((item: TItem) => string | undefined) | undefined,
+  item: TItem,
+  background: string | undefined
+) =>
+  useMemo(() => (backgroundResolver ? backgroundResolver(item) : background), [
```

Comment:
unneccessary `useMemo`?

also if it _is_ needed, put it as close to the public API as possible.

Reply:
Yea, all resolvers that user can inject should be memoized, since we have no control over them. They can be slow.

Reply:
Extracted into own function.